src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowModuleBuilding.java

```diff
@@ -60,6 +59,13 @@ public AbstractWindowModuleBuilding(final B building, final String resource)
         }
     }
 
+    @Override
+    public void setPage(final boolean relative, final int page)
+    {
+        super.setPage(relative, page);
+        mc.getSoundManager().play(SimpleSoundInstance.forUI(SoundEvents.BOOK_PAGE_TURN, 1.0F));
```

Comment:
Move sound to super would make more sense?

Reply:
I didnt put that there intentionally, as I didnt want it to apply to ALL uis, only to those specific ones.